### PR TITLE
beacon: add flag --do-plotting

### PR DESCRIPTION
**Pull request by doyougnu**

### CODE REVIEW

File: beacon/app/beacon.hs [hunk 1]
```diff
@@ -94,22 +94,31 @@ data BenchmarksCompareOptions = BenchmarksCompareOptions {
...
       , overwriteData      :: !OverwriteData
+        -- | Whether to the plot the CSV files that 'db-analyser' produces.
+      , doPlotting         :: !EmitPlots
     }
 
 -- | Should the program overwrite the benchmark data?
 data OverwriteData = OverwriteData | DoNotOverwriteData
 
+-- | Should the program plot the benchmark data?
+data EmitPlots = EmitPlots | DoNotEmitPlots
```
Comment (dnadales) [inline beacon/app/beacon.hs]: If we don't want to repeat the data type name we could rename this to:

```suggestion
data ShouldEmitPlots = EmitPlots | DoNotEmitPlots
```

Comment (dnadales) [reply beacon/app/beacon.hs]: But I'm ok with the original name.

**Review by dnadales — APPROVED** [review]
Thank you for adding this extra option!

**Comment by dnadales** [comment]
There are some CI failures though. Could you address those? I could do it otherwise.

**Comment by doyougnu** [comment]
Yes! Sorry for the late reply I was on vacation most of last week (:turkey:), looks like I didn't propagate a change correctly.

**Review by dnadales — APPROVED** [review]

**Pull request merged**